flow/stopwatch_dl.cc

```diff
@@ -102,7 +102,11 @@ void DlStopwatchVisualizer::Visualize(DlCanvas* canvas,
 
   // Actually draw.
   // Note we use kSrcOver, because some of the colors above have opacity < 1.0.
-  canvas->DrawVertices(painter.IntoVertices(), DlBlendMode::kSrcOver, paint);
+  // The Canvas does not actually hold onto a strong reference to the DlVertices
+  // object, so it needs to be cached on the painter to avoid being deleted
+  // while the backend is trying to read data from it.
+  dl_vertices_ = painter.IntoVertices();
+  canvas->DrawVertices(dl_vertices_, DlBlendMode::kSrcOver, paint);
```

Comment:
Shouldn't the backend manage the lifecycle of the vertices in that case?

Reply:
It does not: https://github.com/flutter/flutter/issues/133797

Reply:
Can we add this GH issue to the line, it would be nice to fix this when we have an API fix :)